src/components/Search/Pill.jsx

```diff
@@ -1,36 +1,37 @@
 import './Pill.less'
 import { connect } from 'react-redux'
 import iconMap from '../iconMap'
+import PropTypes from 'prop-types'
 import React from 'react'
 import { removeFilter } from '../../actions/filter'
 import { SLUG_SEPARATOR } from '../../constants'
 
-// The linter does not detect the use of 'fieldName' which _is_ used.
-// eslint-disable-next-line no-unused-vars
-export const Pill = ( { fieldName, value, trimmed, remove } ) =>
-    <li className="pill flex-fixed">
-      <span className="name">{ trimmed }</span>
-      <button onClick={ remove }>
-        <span className="u-visually-hidden">
-          {'Remove ' + trimmed + ' as a filter'}
-        </span>
-        { iconMap.getIcon( 'delete' ) }
-      </button>
-    </li>
-
-
-export const mapStateToProps = ( state, ownProps ) => {
-  const parts = ownProps.value.split( SLUG_SEPARATOR )
-  const trimmed = parts.length > 1 ? parts.pop() : parts[0];
-
-  return {
-    ...ownProps,
-    trimmed
+export class Pill extends React.Component {
+  render() {
+    const { value, remove } = this.props
+    const parts = value.split( SLUG_SEPARATOR )
+    const trimmed = parts.length > 1 ? parts.pop() : parts[0]
```

Comment:
🤔  Just plain 'ol `pop` might work here.  If it is 0, it will pop to an empty array

Reply:
You're right. Not sure who wrote this, but i was just copy pasting things around.

Had to test it myself here:
<img width="338" alt="Screen Shot 2020-08-06 at 4 54 07 PM" src="https://user-images.githubusercontent.com/1038306/89581757-879ef880-d805-11ea-9c62-f61bdba1f37e.png">

Reply:
oh it was me originally, but you know, older & wiser etc.